Mathlib/RingTheory/LocalProperties/Properties.lean

```diff
@@ -701,9 +416,11 @@ end FiniteType
 
 section IsIntegallyClosed
 
+-- TODO: Prove that `IsIntegrallyClosed` is preserved by localization
```

Comment:
I would even split each of these sections into separate files and shave imports using `#min_imports` to see if that is meaningful or not.